dev/bots/analyze.dart

```diff
@@ -1839,6 +1841,29 @@ Future<void> verifyNullInitializedDebugExpensiveFields(String workingDirectory,
   }
 }
 
+final RegExp tabooPattern = RegExp(r'^ *///.*\b(simply)\b', caseSensitive: false);
```

Comment:
Perhaps start with a List of patterns?

Potential offenders besides simply:
- obviously
- naturally
- of course
- trivially
- merely
- easily

Fillers:
- basically
- really

Redundancies:
- already existing
- completely new

Reply:
BTW, for the exact list of taboo words, maybe a poll on Discord would help to reduce the risk of some folks feeling a doc language autocracy is being born :-).